src/macros.rs

```diff
@@ -131,12 +131,15 @@ macro_rules! implement_vertex {
                                 offset
                             },
                             {
+                                // Obtain the type of the $field_name field of $struct_name and
+                                // call get_type on it.
                                 fn attr_type_of_val<T: $crate::vertex::Attribute>(_: Option<&T>)
                                     -> $crate::vertex::AttributeType
                                 {
                                     <T as $crate::vertex::Attribute>::get_type()
                                 }
-                                attr_type_of_val(None::<&$struct_name>.map(|v| v.$field_name))
+                                let field_option = None::<&$struct_name>.map(|v| v.$field_name);
```

Comment:
Why isn't this `&v.$field_name`?

Reply:
How did this even compile before?^^

Reply:
The macro itself compiled but expansion only happens in the tests which right now always fail on travis. So I didn't notice that the test for `implement_vertex!` failed as well.